src/d3a/d3a_core/simulation.py

```diff
@@ -301,6 +311,30 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
         self.simulation_config.external_redis_communicator.sub_to_aggregator()
         self.simulation_config.external_redis_communicator.start_communication()
         self._update_and_send_results()
+
+        if d3a.constants.IS_CANARY_NETWORK is True:
+            slot_count = sys.maxsize
+            import datetime
+
+            ConstSettings.GeneralSettings.RUN_REAL_TIME = True
```

Comment:
This flag could be removed, since canary network is expected to run in realtime.

Reply:
As discussed, removing this. Thanks